### DISCUSSION TITLE
I have a very strange commit with several participants, please help me understand how it happened?

### DISCUSSION THREAD

**Discussion by sanchezzzhak** [community/community#22796] [New to GitHub]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/e/eae3d10e01a34abfe036e1f39d36cffea2ca035f.png" data-download-href="/uploads/short-url/xvVP1obSoEVCHN4mxCtkhBZ1DP1.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/1337066/181073160-e6e89f62-d1d7-4e3f-965e-bd19ff63b3de.png" alt="image" data-base62-sha1="xvVP1obSoEVCHN4mxCtkhBZ1DP1" width="690" height="165" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/e/eae3d10e01a34abfe036e1f39d36cffea2ca035f_2_690x165.png, https://aws1.discourse-cdn.com/github/optimized/2X/e/eae3d10e01a34abfe036e1f39d36cffea2ca035f_2_1035x247.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/e/eae3d10e01a34abfe036e1f39d36cffea2ca035f_2_1380x330.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/e/eae3d10e01a34abfe036e1f39d36cffea2ca035f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1892×453 75.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<aside class="onebox githubcommit">
  <header class="source">
      <a href="https://github.com/sanchezzzhak/device-detector/commit/14e6ae3ed5d5196fd1252a1a4debc46f847647d1?branch=14e6ae3ed5d5196fd1252a1a4debc46f847647d1&amp;diff=unified" target="_blank" rel="noopener nofollow ugc">github.com/sanchezzzhak/device-detector</a>
  </header>
  <article class="onebox-body">
    <div class="github-row">
  <div class="github-icon-container" title="Commit">
    <svg width="60" height="60" class="github-icon" viewbox="0 0 14 16" aria-hidden="true"><path d="M10.86 7c-.45-1.72-2-3-3.86-3-1.86 0-3.41 1.28-3.86 3H0v2h3.14c.45 1.72 2 3 3.86 3 1.86 0 3.41-1.28 3.86-3H14V7h-3.14zM7 10.2c-1.22 0-2.2-.98-2.2-2.2 0-1.22.98-2.2 2.2-2.2 1.22 0 2.2.98 2.2 2.2 0 1.22-.98 2.2-2.2 2.2z"></path></svg>
  </div>

  <div class="github-info-container">
    <h4>
      <a href="https://github.com/sanchezzzhak/device-detector/commit/14e6ae3ed5d5196fd1252a1a4debc46f847647d1" target="_blank" rel="noopener nofollow ugc">fix(device) merge brand Alldocube into cube</a>
    </h4>

    <div class="github-info">
      <div class="date">
        committed <span class="discourse-local-date" data-format="ll" data-date="2020-10-04" data-time="21:13:05" data-timezone="UTC">09:13PM - 04 Oct 20 UTC</span>
      </div>

      <div class="user">
        <a href="https://github.com/chrisblakley" target="_blank" rel="noopener nofollow ugc">
          <img alt="chrisblakley" src="https://user-images.githubusercontent.com/1337066/181073162-74ad3fd5-dbdc-4e60-bd23-bacda2f5f17d.jpeg" class="onebox-avatar-inline" width="20" height="20">
          chrisblakley
        </a>
        
      </div>

      <div class="lines" title="changed 5 files with 118 additions and 18 deletions">
        <a href="https://github.com/sanchezzzhak/device-detector/commit/14e6ae3ed5d5196fd1252a1a4debc46f847647d1" target="_blank" rel="noopener nofollow ugc">
          <span class="added">+118</span>
          <span class="removed">-18</span>
        </a>
      </div>
    </div>

  </div>
</div>

  <div class="github-row">
    <pre class="github-content" style="white-space: normal;">feat(device) detect brand Cube models: iWork 8 Air (I1-TF, I1-TFB, I1-TFD), iWork 8 Ultimate (I1-TC),
 iWork 10 Pro (i1002S)
issue #6355</pre>
  </div>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by airtower-luna** [comment, marked as answer]
<p>To quote the <a href="https://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History">Git Book chapter “Viewing the Commit History”</a>:</p>
<blockquote>
<p>You may be wondering what the difference is between <em>author</em> and <em>committer</em>. The author is the person who originally wrote the work, whereas the committer is the person who last applied the work. So, if you send in a patch to a project and one of the core members applies the patch, both of you get credit — you as the author, and the core member as the committer.</p>
</blockquote>
<p>A common reason for that when working with Github are rebase merges, where the committer is the person who did the merge. But depending on your git workflow there may be others.</p>
